tests/monitors/solr/solr_test.py

```diff
@@ -3,33 +3,27 @@
 
 import pytest
 
-from helpers.assertions import has_datapoint_with_dim, has_log_message, tcp_socket_open
-from helpers.util import container_ip, run_service, run_agent, wait_for
+from helpers.assertions import has_datapoint_with_dim, has_log_message, tcp_socket_open, has_datapoint_with_metric_name
+from helpers.util import container_ip, run_service, run_agent, wait_for, ensure_always
 
 pytestmark = [pytest.mark.collectd, pytest.mark.etcd, pytest.mark.monitor_with_endpoints]
 
-MONITOR_CONFIG = string.Template(
-    """
-monitors:
-- type: collectd/solr
-  host: $host
-  port: 8983
-  includeMetrics:
-  - solr.core_deleted_docs
-    solr.core_index_size
-  excludeMetrics:
-  - solr.core_max_docs
-    solr.core_num_docs
-"""
-)
-
 
 def test_solr_monitor():
+    monitor_config = string.Template(
```

Comment:
If you are going to put config in the test function itself, just use an f-string and textwrap.dedent like some of the other tests do.